packages/react-native-renderer/src/ReactNativeAttributePayloadFabric.js

```diff
@@ -445,6 +438,29 @@ function diffProperties(
   return updatePayload;
 }
 
+function processProperty(
+  prop: mixed,
+  attributeConfig: AnyAttributeType,
+): mixed {
+  if (typeof attributeConfig !== 'object') {
+    return prop;
+  }
+
+  const process = {attributeConfig};
```

Comment:
```suggestion
  const {process} = attributeConfig;
```

Reply:
🤦‍♀️. We should probably fork the unit tests for this to run against the new one.